### PR TITLE
[[ Documentation ]] Fixes to filename-of-stack.lcdoc

### PR DESCRIPTION
**Pull request by asayd**
- Improved examples.
- Added references.
- Fixed links to referenced tokens.

### CODE REVIEW

File: docs/dictionary/property/filename-of-stack.lcdoc [hunk 1]
```diff
@@ -64,12 +73,14 @@ is empty.
...
 > <main stack> might be "/Volumes/Disk/MyApp.app/Contents/MacOS/MyApp".
 
-References: stacks (function), property (glossary), OS X (glossary),
-main stack (glossary), stack file (glossary), substack (glossary),
-standalone application (glossary), file path (glossary),
-folder (glossary), application bundle (glossary), effective (keyword),
-file (keyword), saveStackRequest (message), stack (object),
-defaultFolder (property), stackFiles (property)
+References: application bundle (glossary), defaultFolder (property),
+defaultFolder (property), effective (keyword), empty (constant),
```
Comment (livecodeali) [inline docs/dictionary/property/filename-of-stack.lcdoc]: There are two instances of `defaultFolder (property)` here

**Comment by asayd** [comment]
@livecodeali Fixed.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 8e72d8b

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 8e72d8bdd387f0abd1277afdb7be0949fc1ef450

**Comment by livecode-vulcan** [comment]
😎 test success 8e72d8bdd387f0abd1277afdb7be0949fc1ef450
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/91)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/91)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/91)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/91)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/91)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/91)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/91)

**Pull request merged**